MdeModulePkg/Application/MpServicesTest/MpServicesTest.c

```diff
@@ -0,0 +1,560 @@
+/** @file
+  UEFI Application to exercise EFI_MP_SERVICES_PROTOCOL.
+
+  Copyright (c) 2022, Qualcomm Innovation Center, Inc. All rights reserved.<BR>
```

Comment:
@kuqin12 if you're online. Is the copyright correct? All Qualcomm changes with no Microsoft copyright? Double checking since that's a little unusual for a contribution in the team.